src/main/java/com/extendedclip/papi/expansion/mcstatistics/StatisticsExpansion.java

```diff
@@ -264,6 +264,10 @@ private String calculateTotal(final OfflinePlayer player, final Statistic statis
                 continue;
             }
 
+            if (material.name().startsWith("LEGACY_")) {

```

Comment:
There exists a `isLegacy()` method, which may be more reliable. Not sure if it exists since the start tho or has been a more recent addition to the API.

Reply:
isLegacy is deprecated with reason "Do not use for any reason.", and just does the same thing anyway
Plus, for compatibility, isLegacy only exists since 1.13 and would end up with NoSuchMethodError